### PR TITLE
Issues/job credentials for deployment

### PR DESCRIPTION
**Pull request by zcgandcomp**

### CODE REVIEW

**Review by banterCZ — APPROVED** [review]
Looks good.

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'java', 'javascript' ]
```
Comment (romanstrobl) [inline .github/workflows/codeql-analysis.yml:31]: This is a Java component, JavaScript should not be required.

**Review by romanstrobl — CHANGES_REQUESTED** [review]
Let's remove the JavaScript scanning, otherwise looks good.

File: .github/workflows/codeql-analysis.yml (hunk 1 above)
Comment (zcgandcomp) [reply .github/workflows/codeql-analysis.yml]: Ok

**Review by romanstrobl — APPROVED** [review]
Looks OK.

**Pull request merged**